apartment.rb

```diff
@@ -1,2 +1,27 @@
 class Apartment
+	attr_reader :rooms, :monthly_rent
+
+	def initialize(args={})
+		@rooms = args.fetch(:rooms, [])
+		@number = args.fetch(:number, nil)
+		@monthly_rent = args.fetch(:monthly_rent, 1000)
+	end
+
+	def total_sqft
+		self.rooms.inject(0) { |total, room| total + room.sqft }
```

Comment:
Great job on this challenge. Code looks clean. Only thing is that it is unnecessary to call self anywhere in this challenge because you have created attr_readers/accessors that create these getters and/or setter methods for you. For example, in this method you could simply call rooms.inject(0) { |total, room| total + room.sqft } and return the same output. @jennysoup